ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -5138,6 +5138,87 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-azure-ovn-shared-vpc,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
This option sticks between `make update` calls, so you'll have to edit it manually.